setup_and_configuration/post-installation-configuration.md

```diff
@@ -2,9 +2,19 @@
 
 ### **Change Heath Center Name**
 
-1. Login and at  the home page select System Administration as shown below![](/assets/healthcentre.png)
+1. Login and at the home page select System Administration as shown below
 
-2. Select Health Center Configurations as shown below.![](/assets/Screen Shot 2019-04-12 at 12.36.50 PM.png)3. On clicking Health Center Configurations. Fill in the Name of the Facility in the form field And Click Save as shown below.![](/assets/Screen Shot 2019-04-12 at 12.42.09 PM.png)4. The name of the health centre shall be displayed as shown below
+    ![click button for next steps to HC name config](/assets/System_administration_button.png)
```

Comment:
@ODORA0 text should be [System Administration]

Reply:
@ssmusoke Done